acquia/cli
# CLI-657: Improve SSH key polling and permission checks

**Motivation**

Fixes #818 

**Proposed changes**

- Add permission checks for git, prod, and non-prod environments (the three places where SSH keys can be installed)
- Warn the user if they lack permissions for any of these
- Poll all three places to make sure SSH keys are fully installed and available

**Testing steps**


0. Use or mimic an IDE (export REMOTEIDE_UUID and AH_SITE_ENVIRONMENT=ide)
1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. Remove any existing local IDE SSH keys
4. Run `acli ide:wizard`
5. Depending on whether you have permissions to access git / prod / non-prod environments in the respective application, see that the corresponding environment is either polled or you are warned that you lack access.

**Merge requirements**
